src/js/helpers/aboutTheDataHelper.js

```diff
@@ -171,4 +190,24 @@ export const convertDatesToMilliseconds = (data) => data.map((datesObj) => {
     return { publication_date: publicationDate.getTime(), certification_date: certificationDate.getTime() };
 });
 
+export const formatUnlinkedDataRows = (data, type) => ([
```

Comment:
I personally like having this as a helper rather than a model. But I wonder if it should be a model to keep in line with our precedents elsewhere in the codebase? It's a relatively small payload so I think we could establish (or further establish) the precedent that we only use models for large API payloads? 

Either way it would be good to have some kind of standard and perhaps even codify  it in some kind of documentation. 

Thoughts?

Reply:
We certainly can move these data points to a model.  There is benefit to the models in the structure, tests, custom fields and reusability (core or base models).  This api endpoint has all fields required and not nullable in the current API contract which is in a comment in this ticket https://federal-spending-transparency.atlassian.net/browse/DEV-6545?focusedCommentId=64146.  And I did not see any use case for custom fields. I feel we should opt in for all models. Would be a good dev meeting topic.

Reply:
Yeah. I'm a bit confused on our standard w/ respect to when we create a model. Would be good to document somewhere.